doc/sphinx/proof-engine/tactics.rst

```diff
@@ -105,6 +108,354 @@ bindings_list`` where ``bindings_list`` may be of two different forms:
   .. exn:: Not the right number of missing arguments.
      :undocumented:
 
+.. _intropatterns:
+
+Intro patterns
+~~~~~~~~~~~~~~
+
+Intro patterns let you specify the name to assign to variables and hypotheses
+introduced by tactics.  They also let you split an introduced hypothesis into
+multiple hypotheses or subgoals.  Common tactics that accept intro patterns
+include :tacn:`apply`, :tacn:`intros` and :tacn:`destruct`.
```

Comment:
It's not `apply` but `apply ... in` which is a very different (and much less known) variant which supports an `as` clause. Thus, I would not cite `apply` in this list. I would rather cite `assert` or `specialize`.

Reply:
I see your point.  My idea for revising doc for `apply` and other tactics is to show the full syntax covering all options first, then identifying the most common options, followed by the details, akin to this section.  So there may only be a single index entry for `apply`.  I'll put in `assert`.